src/reporter.ts

```diff
@@ -241,56 +241,45 @@ class RPReporter implements Reporter {
   }
 
   onTestBegin(test: TestResp): void {
-    //create suite
-    const suiteHasParent = test.parent.parent?._isDescribe;
-    const suiteTitle = suiteHasParent ? test.parent.parent?.title : test.parent.title;
-    if (!this.findTestItem(this.suites, suiteTitle)) {
-      const codeRef = getCodeRef(test, TEST_ITEM_TYPES.SUITE);
+    const suitesOrder: string[] = [];
+    const createSuitesOrder = (suite: any) => {
+      if (!suite?._isDescribe) {
+        return;
+      }
+      suitesOrder.push(suite.title);
+      createSuitesOrder(suite.parent);
+    };
+    createSuitesOrder(test.parent);
```

Comment:
Could you please move this function to the class level? Now it will be created every time during `onTestBegin` execution.